cmd/minishift/cmd/config/config.go

```diff
@@ -100,6 +100,10 @@ var settings []Setting = []Setting{
 		callbacks:   []setFn{RequiresRestartMsg},
 	},
 	{
+		name: "hostfolders-automount",
+		set:  SetBool,
```

Comment:
@LalatenduMohanty actually we are dealing with a resource, which is generally pluralized. I actually wanted to do the same for the actual command...